pkg/v0/cmd.go

```diff
@@ -224,19 +305,33 @@ func detectNewCommits(ctx context.Context, logger *logrus.Entry, stagingDir, cen
 		case flags.HTTPS:
 			remote = "https://github.com/operator-framework/" + repo + ".git"
 		}
-		if _, err := internal.RunCommand(repoLogger, exec.CommandContext(ctx,
+		ref, ok := repoRefs["operator-framework/" + repo]
+		if !ok {
+			logger.Debugf("using 'master' for repo %q", repo)
```

Comment:
Prefer to be explicit. Do explicit defaulting in the caller. If `repo` is not in `repoRefs` here, returning an error would be appropriate.